packages/react/src/Figure/Figure.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react'
+import { createRef } from 'react'
+import { Figure } from './Figure'
+import '@testing-library/jest-dom'
+
+describe('Figure', () => {
+  it('renders', () => {
+    render(<Figure />)
+
+    const component = screen.getByRole('figure')
+
+    expect(component).toBeInTheDocument()
+    expect(component).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
+    render(<Figure />)
+
+    const component = screen.getByRole('figure')
+
+    expect(component).toHaveClass('ams-figure')
+  })
+
+  it('renders an additional class name', () => {
+    render(<Figure className="extra" />)
+
+    const component = screen.getByRole('figure')
+
+    expect(component).toHaveClass('ams-figure extra')
+  })
+
+  it('supports ForwardRef in React', () => {
+    const ref = createRef<HTMLElement>()
+
+    render(<Figure ref={ref} />)
+
+    const component = screen.getByRole('figure')
+
+    expect(ref.current).toBe(component)
+  })
+})
```

Comment:
A `figcaption` is the accessible name for the figure, right? Can we test that? So something like (pseudocode):

```
render(
  <Figure>
    <Figure.Caption>Name</Figure.Caption>
  </Figure>
)

  const component = screen.getByRole('figure', { name: 'Name' })

  expect(component).toBeInTheDocument()

```

Reply:
Doesn’t seem to work, strangely.

Added https://gemeente-amsterdam.atlassian.net/browse/DES-1120.